rpmlint/checks/Allowlisting.py

```diff
@@ -0,0 +1,386 @@
+#############################################################################
+# Author        : Matthias Gerstner
+# Purpose       : reusable code for dealing with security allow lists
+#############################################################################
```

Comment:
The whole comments are moot, just convert them to module docstring if it makes sense.